cli/dcoscli/service/main.py

```diff
@@ -305,4 +306,8 @@ def _log_marathon(follow, lines, ssh_config_file):
 
     emitter.publish(DefaultError("Running `{}`".format(cmd)))
 
+    # In the tests, we SIGKILL this process group, and the output
+    # doesn't get flushed unless we explicitly do so here.
+    sys.stderr.flush()
```

Comment:
We don't need to measure IO buffering to know it has a significant impact.  It can be the difference between disk write and RAM write.  It doesn't matter much right now because our output is so small, but it could in the future, so I'd rather not flush every time.

Leaky abstractions are a bad idea because they make it dangerous to modify the implementation.  That isn't the case here.  It's always safe to flush stderr.

If you really want, I'll surface flush=False through the emitter layer, but I don't think that's a good idea either.